doc/requirements.txt

```diff
@@ -9,12 +9,12 @@ sphinxcontrib-bibtex
 myst-parser  # recommark is deprecated, https://stackoverflow.com/a/71660856/13573820
 sphinx-copybutton
 
-pypsa >=0.24, <0.25
+pypsa >=0.25, <=0.30
 vresutils>=0.3.1
-powerplantmatching>=0.5.5
-atlite>=0.2.9
+powerplantmatching>=0.5.19
```

Comment:
Would you mind to clarify what has happened in ppm `0.5.19`? It's [release notes](https://powerplantmatching.readthedocs.io/en/latest/release-notes.html#version-0-5-19-16-09-2024) are not particularly helpful to get it.

Has it been [this fix](https://github.com/PyPSA/powerplantmatching/pull/202) which replaces origin GEM urls with TUB ones, right? Probably good to keep track of it as the solution is being considered as a temporal one according to the explanations in the associated[ PyPSA-Eur issue](https://github.com/PyPSA/pypsa-eur/pull/1265).

Reply:
These limits have been taken from our existing environment file. I believe it was what you mentioned about the paths

Reply:
Ok, great! May it be worth adding a comment on that to `environment.yaml`? 🙏🏽

Reply:
Done

Reply:
Thank you!